spec.md

```diff
@@ -246,8 +249,9 @@ Policies MUST specify the following fields:
   clients SHOULD cache a policy for up to this value from last policy fetch
   time.
 * rua: Address to which aggregate feedback MAY be sent (comma-separated
-  plain-text list of email addresses, optional). For example,
-  "mailto:postmaster@example.com" from [@!RFC3986].
+  plain-text list of email addresses or an HTTPS endpoint, optional). For example,
```

Comment:
Regarding reporting via HTTP(S): I think we may want this, but the spec doesn't currently say what the protocol for reporting looks like. Is it to just POST to this URI? If so, we should say that.